lib/swaggerValidator/trafficValidator.ts

```diff
@@ -67,6 +85,44 @@ export class TrafficValidator {
 
     this.liveValidator = new LiveValidator(liveValidationOptions);
     await this.liveValidator.initialize();
+
+    const container = inversifyGetContainer();
+    this.loader = inversifyGetInstance(LiveValidatorLoader, {
+      container,
+      fileRoot: liveValidationOptions.directory,
+      ...liveValidationOptions,
+      loadSuppression: Object.keys(apiValidationErrors),
+    });
+
+    // re-set the transform context after set the logging function
+    this.loader.setTransformContext();
+    const swaggerPaths = this.liveValidator.swaggerList;
+
+    while (swaggerPaths.length > 0) {
+      const swaggerPath = swaggerPaths.shift()!;
+      let spec;
+      try {
+        spec = await this.loader.load(pathResolve(swaggerPath));
+      } catch (e) {
+        console.log(e);
+      }
+      if (spec !== undefined) {
+        // Get Swagger - operation mapper.
+        if (this.operationSpecMapper.get(swaggerPath) === undefined) {
+          this.operationSpecMapper.set(swaggerPath, []);
```

Comment:
lowercase the swaggerPath here when add to map then fetch doesn't need to do it again.